tests/x509/test_x509_ext.py

```diff
@@ -6048,6 +6048,14 @@ def test_simple(self, backend):
             sct.entry_type
             == x509.certificate_transparency.LogEntryType.PRE_CERTIFICATE
         )
+        assert (
+            sct.hash_algorithm
+            == x509.certificate_transparency.HashAlgorithm.SHA256
+        )
+        assert (
+            sct.signature_algorithm
+            == x509.certificate_transparency.SignatureAlgorithm.RSA
+        )
```

Comment:
This is currently failing locally for me, although I don't understand why:

```
E       AssertionError: assert <SignatureAlgorithm.ECDSA: 3> == <SignatureAlgorithm.RSA: 1>
E        +  where <SignatureAlgorithm.ECDSA: 3> = <builtins.Sct object at 0x102843cb0>.signature_algorithm
E        +  and   <SignatureAlgorithm.RSA: 1> = <enum 'SignatureAlgorithm'>.RSA
E        +    where <enum 'SignatureAlgorithm'> = <module 'cryptography.x509.certificate_transparency' from '/Users/william/personal/devel/cryptography/src/cryptography/x509/certificate_transparency.py'>.SignatureAlgorithm
E        +      where <module 'cryptography.x509.certificate_transparency' from '/Users/william/personal/devel/cryptography/src/cryptography/x509/certificate_transparency.py'> = x509.certificate_transparency
```

As best I can tell, the cert in question is indeed RSA signed:

```
    Signature Algorithm: sha256WithRSAEncryption
         48:30:7c:30:be:e5:bc:5c:e0:5d:c7:0d:0c:ab:4c:f7:34:5b:
         00:5b:65:ef:91:c5:ed:4c:69:81:e6:c6:af:0a:94:49:e8:b3:
         74:b9:be:8d:25:7f:8a:88:59:83:a3:db:e8:30:3a:e8:60:be:
         e8:f6:2f:81:ad:20:05:a3:78:6b:f1:03:71:2c:b2:5a:5c:3e:
         78:87:c9:49:d2:80:a4:cf:26:6d:cd:ec:cc:f2:9d:8f:5a:c1:
         23:a2:e7:8f:b5:d1:34:00:81:e7:44:ad:12:9c:a0:64:f7:0d:
         1b:dc:23:12:6f:33:9b:73:50:ae:f0:04:ea:66:71:39:d5:de:
         f1:ba:b8:a6:ee:a4:d9:32:12:4c:da:18:bd:1d:e2:cf:2d:57:
         44:06:2b:82:ce:66:2d:36:94:4c:31:0a:9b:42:fb:eb:5d:fb:
         40:a2:1a:bf:5b:dd:2b:2b:8d:92:b3:c9:18:21:ff:05:b1:54:
         da:e5:56:aa:5d:6e:e5:22:1f:f7:26:60:4d:2d:9d:43:03:0e:
         6b:99:ac:ed:5a:56:0e:eb:86:12:ef:3e:1a:9b:97:7d:87:6a:
         eb:17:5a:df:82:ff:4c:73:77:1a:c4:05:0e:de:c2:59:0d:1c:
         a1:61:a8:fe:ec:15:20:bf:c1:d1:a2:d2:47:92:3a:f6:01:23:
         e9:df:56:1f
```

but maybe I'm confusing the cert signature with the SCT signature?

Reply:
Oh, indeed I am (and I didn't notice because of macOS's ancient `openssl`):

```
            CT Precertificate SCTs:
                Signed Certificate Timestamp:
                    Version   : v1 (0x0)
                    Log ID    : A7:CE:4A:4E:62:07:E0:AD:DE:E5:FD:AA:4B:1F:86:76:
                                87:67:B5:D0:02:A5:5D:47:31:0E:7E:67:0A:95:EA:B2
                    Timestamp : Nov 17 01:56:25.396 2016 GMT
                    Extensions: none
                    Signature : ecdsa-with-SHA256
                                30:45:02:21:00:B8:03:AD:34:F6:FC:0F:2C:FF:84:A0:
                                86:E5:D7:CF:5A:F0:0A:07:62:6A:7F:B3:A6:44:64:F1:
                                95:A4:48:45:11:02:20:2F:61:8D:53:1B:6F:4A:B8:0A:
                                67:B2:07:E1:8F:6D:AD:D1:04:4A:5E:B3:89:EF:7C:60:
                                C2:68:53:F9:3D:1F:6D
```